templates/pretalx.service.j2

```diff
@@ -6,7 +6,7 @@ After=network.target
 [Service]
 User={{ pretalx_system_user_prefix }}%i
 Group={{ pretalx_system_user_prefix }}%i
-WorkingDirectory=/home/{{ pretalx_system_user_prefix }}%i/.local/lib/python{{ pretalx_system_python_version }}/site-packages/pretalx
-ExecStart=/home/{{ pretalx_system_user_prefix }}%i/.local/bin/gunicorn --bind unix:/run/gunicorn/pretalx_%i --workers {{ pretalx_service_workers }}  --max-requests {{ pretalx_service_workers_max_requests }}  --max-requests-jitter {{ pretalx_service_workers_max_requests_jitter }} pretalx.wsgi
+WorkingDirectory={{ pretalx_virtualenv }}/lib/python{{ pretalx_system_python_version }}/site-packages/pretalx
```

Comment:
You've removed the parametrisation (`%i`) in this service file, fundamentally changing how it's used. I don't think that's a good idea – while I personally don't run this ansible role anymore (in favour of something already using venvs, albeit more customized ones), changing the behaviour like that doesn't seem great.